src/clj/game/macros.clj

```diff
@@ -47,7 +47,9 @@
             'hq-runnable '(not (:hq (get-in runner [:register :cannot-run-on-server])))
             'rd-runnable '(not (:rd (get-in runner [:register :cannot-run-on-server])))
             'archives-runnable '(not (:archives (get-in runner [:register :cannot-run-on-server])))
-            'tagged '(or (pos? (:tagged runner)) (pos? (:tag runner)))
+            'tagged '(or (pos? (:tagged runner))
```

Comment:
Could change this to call the function, or just remove it as it's just repeated code.

Reply:
Which function?